robottelo/datafactory.py

```diff
@@ -32,32 +33,37 @@ def func_wrapper(*args, **kwargs):
 
 
 @datacheck
-def generate_strings_list(length=None, remove_str=None, bug_id=None):
+def generate_strings_list(length=None, exclude_types=None, bug_id=None,
```

Comment:
@rochacbruno you are right, not sure why I had that in my mind. Never mind then and sorry.